lib/formtools/renderers-cell.js

```diff
@@ -5,7 +5,28 @@ var moment = require('moment'),
     async = require('async');
 
 var dateRenderer = module.exports.date = function dateRenderer (date, record, fieldName, model, callback) {
-    return callback(null, moment(date).format('ddd DD/MM/YYYY'));
+    return callback(null, moment(date).format(linz.get('date format')));
+};
+
+var dateTimeRenderer = module.exports.dateTime = function dateTimeRenderer (date, record, fieldName, model, callback) {
+    return callback(null, moment(date).format(linz.get('datetime format')));
+};
+
+var localDateRenderer = module.exports.localDate = function localDateRenderer (date, record, fieldName, model, callback) {
+
+    if (date instanceof Date) {
+        return callback(null, '<span data-linz-localise-date data-linz-date-format="' + linz.get('date format') + '">' + date.toISOString() + '</span>');
```

Comment:
Rather than span, would it be better to use the [`time`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time) element?